tests/performance/load/package.json

```diff
@@ -6,11 +6,11 @@
     "private": true,
     "scripts": {
         "test": "k6 run tests/testSacScopes.js",
-        "har-to-k6": "har-to-k6",
-        "format:check": "prettier --check .",
-        "format:fix": "prettier --write .",
-        "lint:check": "eslint .",
-        "lint:fix": "eslint --fix ."
+        "har-to-k6": "../node_modules/.bin/har-to-k6",
```

Comment:
Why is this change required? We have moved the whole project to another directory. This should stay the same.

Reply:
Good catch, fixed!